core/modules/mysql/MySqlConnection.cc

```diff
@@ -148,7 +148,9 @@ MySqlConnection::cancel() {
     }
     // KILL QUERY only, not KILL CONNECTION.
     int threadId = mysql_thread_id(_mysql);
-    std::string killSql = boost::str(boost::format("KILL QUERY ") % threadId);
+    std::ostringstream osKill;
+    osKill << "KILL QUERY " << threadId;
+    std::string killSql = osKill.str();
```

Comment:
Why not:

    std::string const killSql = "KILL QUERY " + std::to_string(threadId);

?